fed/api.py

```diff
@@ -209,29 +164,25 @@ def init(
     )
 
     logger.info(f'Started rayfed with {cluster_config}')
-    set_exit_on_failure_sending(exit_on_failure_cross_silo_sending)
-    recv_actor_config = fed_config.ProxyActorConfig(
-        resource_label=cross_silo_recv_resource_label)
+    set_exit_on_failure_sending(cross_silo_comm_config.exit_on_sending_failure)
     # Start recv proxy
     start_recv_proxy(
         cluster=cluster,
         party=party,
         logging_level=logging_level,
         tls_config=tls_config,
-        retry_policy=cross_silo_grpc_retry_policy,
-        actor_config=recv_actor_config
+        retry_policy=cross_silo_comm_config.grpc_retry_policy,
+        actor_config=cross_silo_comm_config
     )
 
-    send_actor_config = fed_config.ProxyActorConfig(
-        resource_label=cross_silo_send_resource_label)
     start_send_proxy(
         cluster=cluster,
         party=party,
         logging_level=logging_level,
         tls_config=tls_config,
-        retry_policy=cross_silo_grpc_retry_policy,
-        max_retries=cross_silo_send_max_retries,
-        actor_config=send_actor_config
+        retry_policy=cross_silo_comm_config.grpc_retry_policy,
+        max_retries=cross_silo_comm_config.proxier_fo_max_retries,
+        actor_config=cross_silo_comm_config
```

Comment:
Maybe we could remove `retry_policy` and `max_retries` here and only keep the `actor_config`? Also `actor_config` could be renamed to just `cross_silo_comm_config`?